internal/pkg/helpers/web/web.go

```diff
@@ -0,0 +1,7 @@
+package web
+
+// Web is a struct with embedded RawRequest and RawClient.
+type Web struct {
```

Comment:
`Web` name is not so good.
How about `HTTP`?
cf. teregraf use http: https://github.com/influxdata/telegraf/tree/master/plugins/inputs/http